src/lib/REST/Input/Parser/ContentTree/LoadExtendedNodeInfoRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\REST\Input\Parser\ContentTree;
+
+use Ibexa\AdminUi\REST\Value\ContentTree\LoadNodeExtendedInfoRequest as LoadNodeExtendedInfoRequestValue;
+use Ibexa\Contracts\Rest\Exceptions;
+use Ibexa\Contracts\Rest\Input\ParsingDispatcher;
+use Ibexa\Rest\Input\BaseParser;
+
+class LoadExtendedNodeInfoRequest extends BaseParser
```

Comment:
1. final & internal
2. Naming: ATM the name suggests we're dealing with a value. How about `LoadExtendedNodeInfoStructParser` (see the next comment for reasons behind "Struct").

Reply:
There is already `Ibexa\AdminUi\REST\Input\Parser\ContentTree\LoadSubtreeRequest` and I decided to follow with this convention.

Reply:
Point 1 is still not resolved. REST parsers are `final` across the entire system which is the case for most of the classes in general.